apps/web/src/pages/templates/components/StepEditorSidebar.tsx

```diff
@@ -62,6 +72,7 @@ export const StepEditorSidebar = ({ children }: { children: ReactNode }) => {
         navigate(basePath);
       }}
       data-test-id="step-editor-sidebar"
+      onSubmit={handleSubmit(onSubmitHandler, onInvalid)}
```

Comment:
Due to this, it was not submitting the form on the editor and hence the tests were failing on update workflow